### PR TITLE
Feature/point intercept

### PR DESCRIPTION
**Pull request by elodszopos**
Initial draft. Please let me know what you think.

### CODE REVIEW

File: packages/point-intercept/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@pie-ui/point-intercept",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.0.1",
+  "description": "An interaction to allow students identify coordinates or plot points on a graph by clicking on the graph.",
+  "main": "src/index.js",
+  "dependencies": {
+    "@pie-lib/charting": "../../../pie-lib/packages/charting",
```
Comment (elodszopos) [inline packages/point-intercept/package.json]: This is until pie-lib gets updated with the respective changes that I'm using here.

Comment (edeustace) [reply packages/point-intercept/package.json]: 👍

File: packages/point-intercept/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "@pie-lib/correct-answer-toggle": "^0.1.9",
```
Comment (edeustace) [inline packages/point-intercept/package.json]: `@pie-lib` deps can be bumped

File: packages/point-intercept/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "@pie-lib/render-ui": "^2.0.6",
+    "debug": "^3.1.0",
+    "material-ui": "1.0.0-beta.27",
+    "material-ui-icons": "^1.0.0-beta.36",
+    "react": "^15.6.1",
```
Comment (edeustace) [inline packages/point-intercept/package.json]: `^16.3.2` now

File: packages/point-intercept/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "version": "0.0.1",
+  "description": "An interaction to allow students identify coordinates or plot points on a graph by clicking on the graph.",
+  "main": "src/index.js",
+  "dependencies": {
+    "@pie-lib/charting": "../../../pie-lib/packages/charting",
+    "@pie-lib/correct-answer-toggle": "^0.1.9",
+    "@pie-lib/render-ui": "^2.0.6",
+    "debug": "^3.1.0",
+    "material-ui": "1.0.0-beta.27",
```
Comment (edeustace) [inline packages/point-intercept/package.json]: `^1.0.0-beta.42` now

File: packages/point-intercept/package.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    "material-ui-icons": "^1.0.0-beta.36",
```
Comment (edeustace) [inline packages/point-intercept/package.json]: `@material-ui/icons@^1.0.0-beta.42` now

File: packages/point-intercept/src/controls.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    onDeleteClick: PropTypes.func.isRequired,
+    disabled: PropTypes.bool.isRequired
+  }
+
+  render() {
+    const { classes, disabled, onDeleteClick } = this.props;
+
+    return (<div>
```
Comment (edeustace) [inline packages/point-intercept/src/controls.jsx]: `div` needed?

**Comment by elodszopos** [comment]
All the feedback has been addressed here. Should be looking and working just fine. Only adjustments that are left to be made need to be made in pie-lib/charting. Of course, once those changes are done, we will need to bump the dependency version here.

**Comment by edeustace** [comment]
Looking good - I'm going to merge and build a version - if you wouldn't mind adding some tests - cheers!

**Pull request merged**